pkgs/development/ocaml-modules/zmq/default.nix

```diff
@@ -1,9 +1,16 @@
-{ stdenv, fetchFromGitHub, buildDunePackage, czmq, stdint }:
+{ stdenv, fetchFromGitHub, ocaml, findlib, dune, czmq, stdint }:
 
-buildDunePackage rec {
-  pname = "zmq";
-  version = "20180726";
+if !stdenv.lib.versionAtLeast ocaml.version "4.03"
```

Comment:
Cannot this version constraint also be applied while still using `buildDunePackage`?